charts/nri-kube-events/templates/deployment.yaml

```diff
@@ -131,8 +131,8 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
       {{- end }}
-      {{- if .Values.affinity }}
+      {{- with include "common.affinity" . }}
       affinity:
-{{ toYaml .Values.affinity | indent 8 }}
+        {{- . | nindent 8 -}}
```

Comment:
```sugestion
         {{- . | nindent 8 }}
```

Reply:
Updated in f956251cb7b1d52ab66d364bd839a2b861952ff7.